organization/views_admin.py

```diff
@@ -42,13 +42,17 @@
 
 logger = wevote_functions.admin.get_logger(__name__)
 
+def organization_analyze_tweets_view(request, organization_we_vote_id):
+
+    org_hashtags = organization_analyze_tweets(organization_we_vote_id)
+    return HttpResponse(org_tweets)
```

Comment:
I think we can just work with the one view "organization_analyze_tweets_view", to make all three of these things happen:
1) Retrieve from Twitter and store locally - organization_retrieve_tweets_from_twitter
2) Retrieve in our own object format - TwitterUserManager.retrieve_tweets_cached_locally()
3) Analyze -  organization_analyze_tweets(organization_we_vote_id, tweet_list) - the tweet_list could be a list of Tweet model objects in our format.

At the end, it could use a redirect, like this:
    return HttpResponseRedirect(reverse('candidate:candidate_list', args=()) +
                                '?google_civic_election_id=' + str(google_civic_election_id) +
                                '&state_code=' + str(state_code))